source/client/process_impl.h

```diff
@@ -46,7 +46,8 @@ class ClusterManagerFactory;
  */
 class ProcessImpl : public Process, public Envoy::Logger::Loggable<Envoy::Logger::Id::main> {
 public:
-  ProcessImpl(const Options& options, Envoy::Event::TimeSystem& time_system);
+  ProcessImpl(const Options& options, Envoy::Event::TimeSystem& time_system,
```

Comment:
Can we add a doc comment? We could also explain the optional process_wide argument and say what happens if it isn't provided.